addons/l10n_be_intrastat/data/l10n_be_intrastat_chart_data.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_region_1" model="l10n_be_intrastat.region">
+    <record id="l10n_be_intrastat.intrastat_region_1" model="account.intrastat.region">
       <field name="code">1</field>
```

Comment:
well, that's IS for sure only valid for Belgium

Reply:
Yeah! That's the reason this is on the `l10n_be_intrastat` module :)